Source/AwsCommonRuntimeKit/http/HttpClientConnectionManager.swift

```diff
@@ -16,30 +17,38 @@ public class HttpClientConnectionManager {
         self.options = options
         self.allocator = allocator
         let shutDownPtr: UnsafeMutablePointer<ShutDownCallbackOptions>? = fromOptionalPointer(ptr: options.shutDownOptions)
+        //Todo: add new http2 options as properties in HttpClientConnectionOptions
```

Comment:
don't worry about it yet

Reply:
Usually prefer a backlog item while adding TODO. 

Could you add "Add support for H2" github issue and add the link here?